contributors.py

```diff
@@ -165,12 +173,11 @@ def get_contributors(
                     continue
 
             # Store the contributor information in a ContributorStats object
+            api_endpoint = ghe.removeprefix("https://") if ghe else "github.com"
```

Comment:
Hello @jmeridth,
It was just a copy paste from here using the same login everywhere since the ghe enterprise URL env variable needs the `https://` prefix to successfully work.
https://github.com/github/contributors/pull/197/files#diff-7bc5062a7419f39938478ec57540f845c31e46fadbd35edfeb7edc92922dc54eR188-R189
I can remove this where not explicitly needed, but it was only to follow the same logic everywhere, if the `https://` is needed it's added right next to the request url

Reply:
Sorry, I don't think I was clear.  Since `ghe` already has the `https://` prefix, as you stated.  That is all good.  We can do this:

```python
api_endpoint = ghe if ghe else "https://github.com"
if start_date and end_date:
    commit_url = f"{api_endpoint}/{repo.full_name}/commits?author={user.login}&since={start_date}&until={end_date}"
else:
    commit_url = f"{api_endpoint}/{repo.full_name}/commits?author={user.login}"
```

wdyt?  We control the alternative https://github.com string.

Reply:
Hello @jmeridth you are correct, I did the necessary fixes [here](https://github.com/github/contributors/pull/197/commits/7f806e2edf4165f451f12d52a955294482f85292) and also added another set of tests since the coverage went down to less than 80% with the changes.
I have left it only in 1 place [here](https://github.com/github/contributors/pull/197/files#diff-7bc5062a7419f39938478ec57540f845c31e46fadbd35edfeb7edc92922dc54eR188-R189), in this case we don't have the `https://`
Thanks